### PR TITLE
Consumer tools FMC

### PR DESCRIPTION
**Pull request by jimmynotjim**
Adding the consumer tools featured menu content for debt collection release tomorrow.

## Additions

- Added Consumer Tools FMC Image

## Changes

- Changed the FMC logic to use the parent page title vs url slug because Consumer Tools and Educational Resources don't have url slugs since they don't have overview pages.

## Testing

- checkout the branch and open the mega menu. Consumer tools should have a new FMC and the Poly Com and About Us FMC should continue to be displayed.

## Screenshots

screen shot 2017-01-11 at 1 13 17 pm

## Notes

- Changes are hidden behind a feature flag. You'll need to enable that to see this change.

## Checklist

* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [x] Passes all existing automated tests
* [x] New functions include new tests
* [x] New functions are documented (with a description, list of inputs, and expected output)
* [x] Placeholder code is flagged
* [x] Visually tested in supported browsers and devices
* [x] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)
* [x] Reviewers requested with the [Assignee tool](https://help.github.com/articles/assigning-issues-and-pull-requests-to-other-github-users/) :arrow_right:

### CODE REVIEW

File: cfgov/jinja2/v1/_includes/organisms/_vars-mega-menu.html [hunk 1]
```diff
@@ -46,9 +46,23 @@
             'replaces four disclosure forms with two new ones. ' +
             'We have resources to help you comply.'
 } %}
+{% if flag_enabled(request, 'DEBT_COLLECTION_RELEASE') %}
+    {% set consumer_tools_content = {
+        'link': {
+            'text': 'Stories about debt collection',
+            'url':  '/yourstory/debtcollection',
```
Comment (Scotchester) [inline cfgov/jinja2/v1/_includes/organisms/_vars-mega-menu.html]: URL should be the canonical URL, `/consumer-tools/everyone-has-a-story/debt-collection/`

Comment (jimmynotjim) [inline cfgov/jinja2/v1/_includes/organisms/_vars-mega-menu.html]: Good catch, fixed.

File: cfgov/jinja2/v1/_includes/organisms/_vars-mega-menu.html [hunk 2]
```diff
@@ -50,7 +50,7 @@
     {% set consumer_tools_content = {
         'link': {
             'text': 'Stories about debt collection',
-            'url':  '/yourstory/debtcollection',
+            'url':  '/consumer-tools/everyone-has-a-story/debtcollection',
```
Comment (Scotchester) [inline cfgov/jinja2/v1/_includes/organisms/_vars-mega-menu.html]: `debt-collection` with a hyphen

Comment (jimmynotjim) [inline cfgov/jinja2/v1/_includes/organisms/_vars-mega-menu.html]: Whoops.

**Comment by Scotchester** [comment]
:shipit:

**Pull request merged**